packages/epo-react-lib/src/components/ResponsiveImage/ResponsiveImage.test.tsx

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
not really sure what tests to write that aren't already covered by the Image component, Jest doesn't have a full renderer otherwise I could check for like calculated aspect ratios